Classes/UIScrollView+InfiniteScroll.m

```diff
@@ -275,13 +291,28 @@ - (void)pb_setContentSize:(CGSize)contentSize {
  *  @return CGFloat
  */
 - (CGFloat)pb_clampContentSizeToFitVisibleBounds:(CGSize)contentSize {
+    
+    CGFloat inset = self.infiniteScrollDirection == UIScrollViewInfiniteScrollDirectionTop ? [self pb_originalTopInset] + self.contentInset.bottom : [self pb_originalBottomInset] + self.contentInset.top;
```

Comment:
Long ternary operations especially when doing math can be misleading. if-else would be better.